tests/ConverterTest.php

```diff
@@ -14,7 +14,7 @@
 if (is_dir(CONV_ROOT.'fluxbb/'))
 	define('PUN_ROOT', CONV_ROOT.'fluxbb/');
 
-require_once 'PHPUnit/Framework/TestCase.php';
+require_once 'PHPUnit/Autoload.php';
```

Comment:
Why this change? This [broke our build](https://travis-ci.org/fluxbb/converter/builds/77185502).